react-app/src/library/services/headset.ts

```diff
@@ -230,17 +229,13 @@ export default class HeadsetService {
         'An error has occurred while trying to establish a connection',
         this.selectedImplementation?.errorCode
       );
-      return `Error occurred while establishing connection`;
-      // return `dummy.connectionStatus.error`
+      return `dummy.connectionStatus.error`
```

Comment:
NIT: what's with these status strings? seems a little weird. I think ideally we'd return some sort of enum.